src/main/java/uk/gov/ons/ctp/response/action/service/ActionService.java

```diff
@@ -295,4 +296,20 @@ public Action updateAction(final Action action) {
     }
     return existingAction;
   }
+
+  public void rerunAction(final Action action) {
```

Comment:
You probably want to mark this function as `@Transactional` so that the transaction can be rolled back if there's an error.